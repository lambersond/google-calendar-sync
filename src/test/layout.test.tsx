import { create } from 'react-test-renderer'
import RootLayout from '@/app/layout'

describe('app/layout', () => {
  it('should match layout snapshot', () => {
    const tree = create(<RootLayout>Testing</RootLayout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
