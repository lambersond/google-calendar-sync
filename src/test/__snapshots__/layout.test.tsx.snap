// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/layout should match layout snapshot 1`] = `
<html
  lang="en"
>
  <body
    className="className"
  >
    <div
      className="fixed top-0 left-0 w-full h-16 shadow-nav bg-zinc-900"
    >
      <div
        className="flex items-center h-full max-w-screen-lg mx-auto px-4 justify-between"
      >
        <a
          className="flex gap-4 items-center"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <svg
            data-testid="CalendarSyncIcon"
            fill="#FFF"
            height={48}
            viewBox="2.5 0.5 19 19.7"
            width={48}
          >
            <path
              d="M 10.2942 5.776 h -2.574 v 2.16 h 2.574 v -2.16 z M 9.7794 1.024 v 0.864 H 5.661 V 1.024 H 4.6314 v 0.864 H 4.1166 c -0.5714 0 -1.0245 0.3888 -1.0245 0.864 L 3.087 8.8 c 0 0.4752 0.4582 0.864 1.0296 0.864 h 7.2072 c 0.5663 0 1.0296 -0.3888 1.0296 -0.864 V 2.752 c 0 -0.4752 -0.4633 -0.864 -1.0296 -0.864 h -0.5148 V 1.024 h -1.0296 z m 1.5444 7.776 H 4.1166 V 4.048 h 7.2072 v 4.752 z M 4 11 L 6 11 C 6 12.77 6.481 15.401 8 15 V 13 h 2 v 6 H 4 v -2 h 2.73 C 5.06 16.54 4 14.4 4 12 z M 20.4 4 h -2.73 c 1.43 1.26 2.41 3.01 2.66 5 h -1.93 c -0.23 -1.36 -0.93 -2.55 -1.91 -3.44 V 8 h -2 V 2 h 6 v 0 z M 18.8942 15.776 h -2.574 v 2.16 h 2.574 v -2.16 z M 18.3794 11.024 v 0.864 H 14.261 V 11.024 H 13.2314 v 0.864 H 12.7166 c -0.5714 0 -1.0245 0.3888 -1.0245 0.864 L 11.687 18.8 c 0 0.4752 0.4582 0.864 1.0296 0.864 h 7.2072 c 0.5663 0 1.0296 -0.3888 1.0296 -0.864 V 12.752 c 0 -0.4752 -0.4633 -0.864 -1.0296 -0.864 h -0.5148 V 11.024 h -1.0296 z m 1.5444 7.776 H 12.7166 V 14.048 h 7.2072 v 4.752 z"
            />
          </svg>
          <p
            className="text-2xl font-bold text-slate-200"
          >
            Google Calendar Sync
          </p>
        </a>
        <a
          className="flex items-center gap-2"
          href="/docs"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <svg
            data-testid="DocsIcon"
            fill="#FFF"
            height={23}
            viewBox="0 0 16 15"
            width={24}
          >
            <path
              clip-rule="evenodd"
              d="M0 2.5C0 1.96957 0.210714 1.46086 0.585786 1.08579C0.960859 0.710714 1.46957 0.5 2 0.5H14C14.5304 0.5 15.0391 0.710714 15.4142 1.08579C15.7893 1.46086 16 1.96957 16 2.5V12.5C16 13.0304 15.7893 13.5391 15.4142 13.9142C15.0391 14.2893 14.5304 14.5 14 14.5H2C1.46957 14.5 0.960859 14.2893 0.585786 13.9142C0.210714 13.5391 0 13.0304 0 12.5V2.5ZM3.293 3.793C3.48053 3.60553 3.73484 3.50021 4 3.50021C4.26516 3.50021 4.51947 3.60553 4.707 3.793L7.707 6.793C7.89447 6.98053 7.99979 7.23484 7.99979 7.5C7.99979 7.76516 7.89447 8.01947 7.707 8.207L4.707 11.207C4.5184 11.3892 4.2658 11.49 4.0036 11.4877C3.7414 11.4854 3.49059 11.3802 3.30518 11.1948C3.11977 11.0094 3.0146 10.7586 3.01233 10.4964C3.01005 10.2342 3.11084 9.9816 3.293 9.793L5.586 7.5L3.293 5.207C3.10553 5.01947 3.00021 4.76516 3.00021 4.5C3.00021 4.23484 3.10553 3.98053 3.293 3.793ZM9 9.5C8.73478 9.5 8.48043 9.60536 8.29289 9.79289C8.10536 9.98043 8 10.2348 8 10.5C8 10.7652 8.10536 11.0196 8.29289 11.2071C8.48043 11.3946 8.73478 11.5 9 11.5H12C12.2652 11.5 12.5196 11.3946 12.7071 11.2071C12.8946 11.0196 13 10.7652 13 10.5C13 10.2348 12.8946 9.98043 12.7071 9.79289C12.5196 9.60536 12.2652 9.5 12 9.5H9Z"
              fill-rule="evenodd"
            />
          </svg>
          <p
            className="text-lg font-bold text-slate-200"
          >
            Docs
          </p>
        </a>
      </div>
    </div>
    <main
      className="flex min-h-screen min-w-screen flex-col items-center py-24 gap-8"
    >
      Testing
    </main>
  </body>
</html>
`;
